server/src/core/api-rest/api-rest-query-builder/factories/find-many-query.factory.ts

```diff
@@ -6,18 +6,19 @@ import { mapFieldMetadataToGraphqlQuery } from 'src/core/api-rest/api-rest-query
 @Injectable()
 export class FindManyQueryFactory {
   create(objectMetadata, depth?: number): string {
+    const objectNameSingular = capitalize(
+      objectMetadata.objectMetadataItem.nameSingular,
+    );
+    const objectNamePlural = objectMetadata.objectMetadataItem.nameSingular;
```

Comment:
this looks suspicious! also we should capitalize it!

Reply:
🤦‍♂️